src/common/pool_map.c

```diff
@@ -2113,6 +2115,427 @@ pool_map_update_failed_cnt(struct pool_map *map)
 	return 0;
 }
 
+#define PMAP_VER_MAX		((uint32_t)-1)
+#define PMAP_FAIL_INLINE_NR	(8)
+struct pmap_fail_ver {
+	uint32_t		 pf_start_ver;
+	uint32_t		 pf_end_ver;
+};
+
+struct pmap_fail_node {
```

Comment:
so this is assuming node is the fault domain, although this could be false if user chose a different domain type?

Reply:
yes, i thought about it. If using RACK level domain type, we may need firstly check tgt level fail pm_ver ranges, and merge to node level, and then merge to RACK level. and compare overlapped ranges between different RACKs.
As currently the actual domain level is node, so I just handle it at node level now. Can handle RACK level later when needed.